pyproject.toml

```diff
@@ -5,28 +5,28 @@ description = ""
 authors = ["CPR Tech <tech@climatepolicyradar.org>"]
 
 [tool.poetry.dependencies]
-python = "^3.9"
-sentence-transformers = "^2.2.0"
-huggingface_hub = ">=0.14.0,<1.0.0"
-click = "^8.0.4"
-pytest = "^7.1.1"
-pydantic = "^2.4.0"
-python-json-logger = "^2.0.4"
-awscli = "^1.27.8"
-boto3 = "^1.26.153"
+python = "^3.10"
+sentence-transformers = "^2.2.2"
+huggingface_hub = "^0.24.6"
+click = "^8.1.7"
+pytest = "^8.3.2"
+pydantic = "^2.8.2"
+python-json-logger = "^2.0.7"
+awscli = "^1.34.10"
+boto3 = "^1.35.10"
 aws-error-utils = "^2.7.0"
-moto = "^4.1.11"
-torch = "2.0.0"
+moto = "^5.0.13"
```

Comment:
Major but a testing library so not core functionality (thumbs up)

Reply:
👌🏻